astroquery/dace/tests/test_dace_remote.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from astropy.tests.helper import remote_data
+from astroquery.dace import Dace
+
+
+@remote_data
+class TestDaceClass(unittest.TestCase):
+
+    def test_should_get_radial_velocities(self):
+        radial_velocities_table = Dace.query_radial_velocities('HD40307')
+        assert radial_velocities_table is not None and 'rv' in radial_velocities_table.colnames
```

Comment:
can you add one more assert here, to check reasonable number of observations returned in the table? Currently an empty table passes this test.

Reply:
You are right. I will add a more meaningful test.